index.d.ts

```diff
@@ -58,10 +67,12 @@ export interface Subscription {
   unsubscribe: () => void;
 }
 
-export interface Message {
-  command: string;
-  body: string;
-  headers: ExtendedHeaders,
+export interface SubscriptionsMap {
+  [id: string]: (frame?: Frame) => any;
```

Comment:
Why `frame` as an optional parameter?

Reply:
According to https://github.com/JSteunou/webstomp-client/blob/master/src/client.js#L271 the `callback` has `message` as parameter, so the user can call `ack` or `nack` on the message.

Reply:
My mistake, I copied the callback of the connection, and as you have said this should be the subscription callback: `(message: Message) => any`.